src/main/java/seedu/address/model/service/Vehicle.java

```diff
@@ -30,8 +29,8 @@ public class Vehicle {
      * @param brand       This is the brand of the vehicle (i.e. KIA).
      * @param type        What vehicle type this vehicle is (i.e. Bike)
      */
-    public Vehicle(int ownerId, String plateNumber, String color, String brand, VehicleType type) {
-        id = ++incrementalID;
+    public Vehicle(int id, int ownerId, String plateNumber, String color, String brand, VehicleType type) {
+        this.id = id;
```

Comment:
hmm specifying the ID eh...

Reply:
hmm ideally id shldnt be decided during runtime like incrementing a counter to act as the id (assuming it increments when we insert vehicles to a list) bcs we need to persist those ids in alot of nested arrays in Customers and Appointments when its saved to the json

we probably just need some helper class to generate unique ids when generating new objects that require id